### PR TITLE
Move settings button from header to Swap menu.

### PR DESCRIPTION
**Pull request by memacius32**
## Description
- Moved Settings button from Header into Swap menu.
- Separate Settings styles.
- Reorder and group Settings imports. 

## Affected components
- Header
- Settings
- 
## Affected views
- Swap

## Browsers / viewports verified
- Chrome v96 (Mac os)
- Firefox v95 (Mac os)
- Responsive

### CODE REVIEW

File: src/components/Settings/Settings.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  background-color: ${({ theme }) => theme.bg3};
+`
+
+export const ModalContentWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 2rem 0;
+  background-color: ${({ theme }) => theme.bg2};
+  border-radius: 20px;
```
Comment (specialist0x) [inline src/components/Settings/Settings.styles.tsx:47]:
```suggestion
  border-radius: 10px;
```

I think new designs are using 10px now

Comment (specialist0x) [reply src/components/Settings/Settings.styles.tsx:47]: Or they should be*

**Comment by specialist0x** [comment]
Can you modify this PR to go on to `main`?

Since `b/redesign` has been merged with master, all subsequent PRs can be branched off `main` now 👌🏽

**Comment by specialist0x** [comment]
Due to the new location, do you think we could add a border to the settings menu so that it doesn't blend in with the `Swap` container's background:

image

File: src/components/Settings/Settings.styles.tsx (hunk 1 above)
Comment (memacius32) [reply src/components/Settings/Settings.styles.tsx:47]: This borderRadius you are commenting is just for expert mode toggle Modal, you want to change borderradius to 10px  here and where else?

**Comment by memacius32** [comment]
Totally. what do you think of this? 
Screen Shot 2022-01-05 at 20 03 03

**Review by specialist0x — APPROVED** [review]

**Pull request merged**